backend/internal/handlers/classrooms/assignments/works/works.go

```diff
@@ -1,21 +1,55 @@
 package works
 
 import (
+	"fmt"
+	"strconv"
+
+	"github.com/CamPlume1/khoury-classroom/internal/errs"
 	"github.com/gofiber/fiber/v2"
 )
 
 // Returns the student works for an assignment.
 func (s *WorkService) getWorks() fiber.Handler {
 	return func(c *fiber.Ctx) error {
-		// Implement logic here
-		return c.SendStatus(fiber.StatusNotImplemented)
+		classroomID, err := strconv.Atoi(c.Params("classroom_id"))
+		if err != nil {
+			return errs.BadRequest(err)
+		}
+		assignmentID, err := strconv.Atoi(c.Params("assignment_id"))
+		if err != nil {
+			return errs.BadRequest(err)
+		}
+
+		works, err := s.store.GetWorks(c.Context(), classroomID, assignmentID)
+		if err != nil {
+			fmt.Print(err)
+			return errs.InternalServerError()
+		}
+		return c.Status(200).JSON(works)
```

Comment:
to stay consistent w other calls me and nick landed on having a title for the return, like 
return c.Status(http.StatusOK).JSON(fiber.Map{
			"assignment_outline": assignment,
})

Reply:
done. thanks for picking a convention and sticking to it 👍